crates/forge/tests/integration/spoof.rs

```diff
@@ -0,0 +1,240 @@
+use crate::integration::common::runner::Contract;
```

Comment:
We should add a case where we cancel some entries with a `None` (and a `.default()` object)